shapes/produce_shapes_2017.py

```diff
@@ -96,10 +96,9 @@ def parse_arguments():
         "--binning", required=True, type=str, help="Binning configuration.")
     parser.add_argument(
         "--channels",
-        default=[],
-        nargs='+',
-        type=str,
-        help="Channels to be considered.")
+        default=["em","et","mt","tt"],
```

Comment:
The default needs to be [], otherwise the creation of control shapes (plotting of input variables) will not work

Reply:
done